src/main/java/com/axiomq/starwars/repositories/VoteRepository.java

```diff
@@ -2,6 +2,12 @@
 
 import com.axiomq.starwars.entities.Vote;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
 
 public interface VoteRepository extends JpaRepository<Vote, Long> {
+
+    @Query(
+            value = "SELECT COUNT(DISTINCT v.user_id) FROM vote v WHERE v.character_id = ?1",
+            nativeQuery = true)
+    Integer countDistinctUsers(Long characterId);
```

Comment:
you could use `long countDistinctByUser(User user);` instead native query

Reply:
how would i know if the returned number of users is only for my character's vote, not from the others ? that's why i need the 
'v.character_id = ?1', right ?

Reply:
long countDistinctUserByCharacter(Character character);